hummingbot/connector/gateway/gateway_price_shim.py

```diff
@@ -58,7 +58,7 @@ class GatewayPriceShim:
     ```
     from hummingbot.connector.gateway_price_shim import GatewayPriceShim
     from decimal import Decimal
-    GatewayPriceShim.get_instance().apply_price_delta("uniswap", "ethereum", "kovan", "WETH-DAI", Decimal(40))
+    GatewayPriceShim.get_instance().apply_price_delta("amm", "ethereum", "kovan", "WETH-DAI", Decimal(40))
```

Comment:
We have connector / exchange named uniswap
https://hummingbot.org/exchanges/uniswap/

this changes here not needed